### ISSUE TITLE
React won't work behind proxy with basic authentication

### ISSUE THREAD

**Issue by barravi** [facebook/react#9804]
Hello

I have a react application running in our local intranet.
All our apps run behind a web proxy with a Basic Authentication, user/password checked against our AD server.

When accessing my react app directly, or via unauthenticated proxy, everything works fine.

When I access the app with the proxy authentication enabled, I manage to fetch the very first page, but every other request performed using 'fetch' fails with a 401 Unauthorized.

Basically the Authorization Header with the hashed credentials is not passed to the fetch request, while it works seamlessly for any other <script> or <css> request.

I looked on the internet for a workaround, but I couldn't find anything other than hardcoding the Authorization Header in the fetch request, which is really not a feasible solution. Nor is implementing specific login logic to work around the proxy issue.

Could you please advise?

**Comment by gaearon** [comment]
This doesn't sound like a React issue (React isn't concerned with network requests), and more like a question about single page apps. I encourage you to ask if on StackOverflow since we use the issue tracker for bugs and features in React, and try to keep it focused.
